spec/requests/portfolios_rbac_spec.rb

```diff
@@ -7,7 +7,7 @@
   let(:block_access_obj) { instance_double(RBAC::Access, :accessible? => false) }
 
   describe "GET /portfolios" do
-    xit 'returns status code 200' do
+    it 'returns status code 200' do
```

Comment:
It seems a little off that with rbac enabled per tenant, we didn't have to validate the attribute was set in this tenant to get a 200 back